unsymlink-lib

```diff
@@ -193,7 +197,8 @@ class MigrationState(object):
                 # files contained within them
                 if details[0] == 'dir':
                     continue
-                f = os.path.join(self.eroot, f.encode(_encodings['fs']).lstrip(b'/'))
+                f = f.encode(_encodings['fs'])
```

Comment:
Are you really, really sure we don't need stripping?

Reply:
Ah, I should put back stripping.